### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by psychoactiv**
### Snackbar
Created fifferent types of snackbar which appear on click of the butto
13 02 2022_04 11 17_REC
13 02 2022_04 10 29_REC
13 02 2022_04 11 17_REC

### CODE REVIEW

File: components/Snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+/* snackbar styling */
+.snacked {
+  position: fixed;
+  text-align: center;
+  background-color: #333;
+  color: #fff;
+  border-radius: 2px;
+  display: flex;
+  flex-wrap: wrap;
+  opacity: 0.94;
```
Comment (tanishk15sharma) [inline components/Snackbar/snackbar.css:17]: try giving `0.9,0.8,0.7 ` something like specific values

File: components/Snackbar/snackbar.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
+@import url("/Utilities/utilities.css");
+/* simple snack bar */
+* {
```
Comment (tanishk15sharma) [inline components/Snackbar/snackbar.css]: you can add this to in text utilities

**Review by tanishk15sharma — APPROVED** [review]
majorly your code is correct

**Comment by Dev-AkshatGupta** [comment]
There is bugs on snack bar please check them Like left side snack bar hiding Check it once

**Comment by psychoactiv** [comment]
> 
did the changes please approve

**Pull request merged**